app/static/src/app/store/run/mutations.ts

```diff
@@ -62,6 +66,14 @@ export const mutations: MutationTree<RunState> = {
         }
     },
 
+    [RunMutation.UpdateAdvancedSettings](state: RunState, payload: SetAdvancedSettingPayload) {
+        state.advancedSettings[payload.option].val = payload.newVal;
+        state.runRequired = {
+            ...state.runRequired,
+            advancedSettingsChanged: true
```

Comment:
an update is needed in `app/static/src/app/components/run/support.ts` to check for this, and then in the user strings to render a nice reason for the plot being out of date. Otherwise you get

> Plot is out of date: unknown reasons, contact the administrator, as this is unexpected. Run model to update.

which is quite alarming

Reply:
done